Extensions/Multiplayer/multiplayertools.ts

```diff
@@ -66,15 +65,18 @@ namespace gdjs {
         gdjs.multiplayerMessageManager.handleUpdateSceneMessagesReceived(
           runtimeScene
         );
-        gdjs.multiplayerMessageManager.handleHeartbeatsToSend();
-        gdjs.multiplayerMessageManager.handleDisconnectedPeers(runtimeScene);
       }
     );
 
     gdjs.registerRuntimeScenePostEventsCallback(
       (runtimeScene: gdjs.RuntimeScene) => {
         if (disableMultiplayerForTesting) return;
 
+        handleLeavingPlayer(runtimeScene);
+        handleJoiningPlayer(runtimeScene);
+        // Joining players are handled in the heartbeats directly.
+        gdjs.multiplayerMessageManager.handleHeartbeatsReceived();
```

Comment:
I don't understand this comment telling that we handled joining players in the heartbeats, while we just before have a function being called that is named "handleJoiningPlayer".

Reply:
yup updating